x86_64/src/vm.rs

```diff
@@ -5,13 +5,14 @@
 // license that can be found in the LICENSE file or at
 // https://opensource.org/licenses/MIT.
 
-/// # x86_64 recursive page map implementation
-///
-/// Hypatia uses recursive page tables with side-loading for
-/// address space inspection and manipulation.
+//! # x86_64 recursive page map implementation
```

Comment:
Can you bust up these changes into another commit?

Reply:
Done.